src/DynamoCoreWpf/Views/Core/NodeView.xaml.cs

```diff
@@ -486,6 +495,24 @@ private void ExpandPreviewControl()
             }
         }
 
+        /// <summary>
+        /// Sets ZIndex of node the maximum value.
+        /// </summary>
+        private void BringInFront()
+        {
+            if (IsMouseOver || PreviewControl.IsMouseOver || DynCmd.IsTestMode)
+            {
+                if (NodeViewModel.StaticZIndex == Int32.MaxValue)
+                {
+                    PrepareZIndex();
+                }
+                var index = ++NodeViewModel.StaticZIndex;
```

Comment:
No, it was NodeViewModel.StaticZIndex++ and I changed to ++NodeViewModel.StaticZIndex. It's line 324 in NodeView.xaml.cs : https://github.com/DynamoDS/Dynamo/pull/6091/files#diff-69be354bfe562b333ee34dec25c9bfd2L324